frontend/src/metabase/public/components/widgets/EmbedModalContent.unit.spec.tsx

```diff
@@ -78,8 +78,14 @@ describe("EmbedModalContent", () => {
     );
 
     openEmbedModal();
-    expect(screen.getByText("My param")).toBeInTheDocument();
-    expect(screen.getByLabelText("My param")).toHaveTextContent("Locked");
+    const parametersSection = screen
```

Comment:
This fixes the signed dashboard embeddings e.g. in this screenshot
<img width="401" alt="image" src="https://user-images.githubusercontent.com/1937582/231563092-659dec93-4b63-4be1-a71d-d2e8f93142a5.png">

both dropdowns with a display label **Source w/ default** in both sections have the same label semantically, so `getByLabelText(label)` would find 2 elements.

This is the change introduced in this PR to make the parameter widget more accessible and testable. See the file `frontend/src/metabase/parameters/components/ParameterValueWidget.jsx`